TYPO3.Neos/Classes/TYPO3/Neos/Command/SiteCommandController.php

```diff
@@ -63,6 +67,89 @@ class SiteCommandController extends CommandController
     protected $systemLogger;
 
     /**
+     * @Flow\Inject
+     * @var ContextFactoryInterface
+     */
+    protected $nodeContextFactory;
+
+    /**
+     * @Flow\Inject
+     * @var NodeTypeManager
+     */
+    protected $nodeTypeManager;
+
+    /**
+     * @Flow\Inject
+     * @var NodeService
+     */
+    protected $nodeService;
+
+    /**
+     * Create a new (blank) site
+     *
+     * This command allows to create a blank site with just a single empty document. The name of the site and the
+     * packageKey to assign to it must be specified.
+     *
+     * If no "--nodeName" is specified the command will create a unique node-name from the name of the site. If a node name is given
+     * it has to be unique.
+     *
+     * If no "--nodeType" is specified the command will check wether it exists and has the superType "TYPO3.Neos:Document".
+     * Without a definition "TYPO3.Neos.NodeTypes:Page" is used as default.
+     *
+     * If the flag "--active" is set the created site will be activated immediately.
+     *
+     * @param string $name The name of the site
+     * @param string $packageKey The site package
+     * @param string $nodeName The name of the site node. If no nodeName is given it will be determined from the siteName.
+     * @param string $nodeType The nodetype to use for the site node.
+     * @param boolean $active Activate the new site immediately.
+     * @return void
+     */
+    public function createCommand($name, $packageKey, $nodeName = NULL, $nodeType = 'TYPO3.Neos.NodeTypes:Page', $active = FALSE)
```

Comment:
@johannessteu i understand your point but the problem is that the only non-abstract documentNode that is part of TYPO3.Neos is TYPO3.Neos:Shortcut which is also no good default for en empty site. 

On the other hand the command would work without Neos.NodeTypes but it would require that a correct NodeType is given.